classes/options-general.php

```diff
@@ -901,11 +901,11 @@ class="radio-thumbnail<?php if ( 'TERRAIN' == $map_type ) {
 
 									<div class="postbox">
 
-										<h3><?php _e( 'Delete SimpleMap Data', 'simplemap' ); ?></h3>
+										<h3><?php _e( 'Delete SimpleMap Data', 'SimpleMap' ); ?></h3>
 
 										<div class="inside">
 											<p class="sub"><span
-													style="color: red;"><?php _e( 'CAUTION! Uninstalling SimpleMap will completely delete all current locations, categories, tags and options. This is irreversible.', 'simplemap' ); ?></span>
+													style="color: red;"><?php _e( 'CAUTION! Uninstalling SimpleMap will completely delete all current locations, categories, tags and options. This is irreversible.', 'SimpleMap' ); ?></span>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 239 characters

<!-- Comment by [SideCI](https://sideci.com) -->